Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInRequest.java

```diff
@@ -161,6 +172,19 @@ public DropInRequest disableVenmo() {
     }
 
     /**
+     * If 3D Secure has been enabled in the control panel and an amount is specified in
+     * {@link DropInRequest#amount(String)}, Drop-In will request a 3D Secure verification for
+     * any new cards added by the user.
+     *
+     * @param requestThreeDSecure {@code true} to request a 3D Secure verification as part of Drop-In,
+     * {@code false} to not request a 3D Secure verification. Defaults to {@code false}.
+     */
+    public DropInRequest requestThreeDSecureVerification(boolean requestThreeDSecure) {
```

Comment:
It is a one time use object, however to stay more in line with the other methods for disabling and enabling things I went with a boolean here. `mAmount` was a holdover from the previous version of Drop-in that was not completely removed in the new version (getters and setters were removed, but the variable was missed). Amount was previously used for display in the UI and could still be used in the future for UI or other purposes.